dbms/src/Flash/Coprocessor/AggregationInterpreterHelper.h

```diff
@@ -33,12 +33,13 @@ bool isFinalAgg(const tipb::Aggregation & aggregation);
 
 bool isGroupByCollationSensitive(const Context & context);
 
-Aggregator::Params buildParams(
+std::shared_ptr<Aggregator::Params> buildParams(
```

Comment:
Because `Aggregator::Params` has no default copy assign operator(because it has two const member), and the benchmark needs store `Params` as member [here](https://github.com/pingcap/tiflash/pull/8985/files/c312346cd19e3a8dd570b17da5d1660295bd1118#diff-fe1f01075b1a6eb635a2258522ac620ca04e669ff64edcc4fd856164b8887cc8R272)

Reply:
Can you try declaring deleted copy/assign operator and defaulted move/assign operator to mitigate this?

Reply:
done

Reply:
What is done?

Reply:
Since `Params` needs to be store in `bencharmrk` class as member, and will be copy assign in `SetUp`. But copy assign/move assign operator is not allowed for `Params`. So we have to make this function return shared_pointer.

So I just declare the copy assign/move assign operator as deleted. Basically nothing changed.